### PR TITLE
RDKTV-11503 : Gdial Crash fix

### PR DESCRIPTION
**Pull request by akhilsarada**
Reason for change: quit thread when geting SIGINT
Test Procedure: Refer JIRA
Risks: Low
Signed-off-by: Akhil Baby Sarada <Akhil_Sarada@comcast.com>

### CODE REVIEW

File: server/main.c [hunk 1]
```diff
@@ -163,11 +164,19 @@ static void gdial_http_server_throttle_callback(SoupServer *server,
   soup_message_set_status(msg, SOUP_STATUS_NOT_FOUND);
 }
 
+static void gdial_quit_app(int signum)
+{
+  g_print("Exiting DIAL Protocol \r\n");
```
Comment (apatel859) [inline server/main.c:169]: DAIL server

File: server/main.c [hunk 2]
```diff
@@ -163,11 +164,19 @@ static void gdial_http_server_throttle_callback(SoupServer *server,
...
+  g_main_loop_quit(loop_);
+  signal(SIGINT,SIG_DFL);
```
Comment (apatel859) [inline server/main.c:171]: remove

File: server/main.c (hunk 1 above)
Comment (akhilsarada) [reply server/main.c]: updated

File: server/main.c (hunk 2 above)
Comment (akhilsarada) [reply server/main.c]: updated

**Comment by scthunderbolt** [comment]
Closed due to inactivity, please restore if still needed.

**Pull request closed without merging**